include/openenclave/bits/log-enc.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef LOG_ENC_H
+#define LOG_ENC_H
+
+#include <openenclave/enclave.h>
+#include <openenclave/bits/log-common.h>
+
+OE_EXTERNC_BEGIN
+
+#define oe_log_trace(module, fmt, ...) oe_log(LOG_TRACE, module, fmt, __VA_ARGS__)
```

Comment:
Variadic macros are a C99 construct.
Our goal has been to stick to C89/90 standard in the public interface since the clients maybe compiling for platforms with old compilers.
Variadic macros are elegant for this purpose, but consider an alternative.